src/mon/ConfigMonitor.cc

```diff
@@ -366,6 +366,7 @@ bool ConfigMonitor::preprocess_command(MonOpRequestRef op)
 	tbl.define_column("OPTION", TextTable::LEFT, TextTable::LEFT);
 	tbl.define_column("VALUE", TextTable::LEFT, TextTable::LEFT);
 	tbl.define_column("RO", TextTable::LEFT, TextTable::LEFT);
+	tbl.define_column("PROFILE", TextTable::LEFT, TextTable::LEFT);
```

Comment:
seems like this should be everywhere we dump them, e.g. config dump as well

Reply:
added a patch to do this!